tests/testcases/externals-link/node_modules/d3-drag

```diff
@@ -0,0 +1 @@
+../../../d3-drag
```

Comment:
any specific reason for this link? I would rather like to keep the testcase self contained.

Reply:
lol nevermind, I should have read the PR description, lol.

Reply:
unfortunately, it has to exist outside the root directory of the test case for the error to occur :(